release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -116,6 +116,12 @@ For this release, Red Hat only supports using MetalLB in layer 2 mode.
 
 For more information, see xref:../networking/metallb/about-metallb.adoc#about-metallb[About MetalLB and the MetalLB Operator].
 
+[id="ocp-4-9-networking-CNI-VRF-plug-in"]
+==== CNI VRF plug-in
```

Comment:
I referenced previous versions, 4.7 & 4.8, of the release notes, and when a feature is generally available it is included in the title. So I would rename this to 'CNI VRF plug-in is generally available'